src/test/java/com/scalar/database/storage/cassandra4driver/SelectStatementHandlerTest.java

```diff
@@ -0,0 +1,489 @@
+    scan = prepareScan();
+    scan.withStart(
+            new Key(new TextValue(ANY_NAME_2, ANY_TEXT_2), new TextValue(ANY_NAME_3, ANY_TEXT_3)))
+        .withEnd(
+            new Key(new TextValue(ANY_NAME_2, ANY_TEXT_2), new TextValue(ANY_NAME_3, ANY_TEXT_4)));
+
+    // Act
+    handler.bind(prepared, scan);
+
+    // Assert
+    verify(builder).setString(0, ANY_TEXT_1);
+    verify(builder).setString(1, ANY_TEXT_2);
+    verify(builder).setString(2, ANY_TEXT_3);
+    verify(builder).setString(3, ANY_TEXT_2);
+    verify(builder).setString(4, ANY_TEXT_4);
+  }
+
+  @Test
+  public void setConsistency_GetOperationWithStrongConsistencyGiven_ShouldPrepareWithQuorum() {
+    // Arrange
+    configureBehavior(null);
+    get = prepareGetWithClusteringKey();
+    get.withConsistency(Consistency.SEQUENTIAL);
+
+    // Act
+    handler.setConsistency(builder, get);
+
+    // Assert
+    verify(builder).setConsistencyLevel(ConsistencyLevel.QUORUM);
+  }
+
+  @Test
+  public void setConsistency_GetOperationWithEventualConsistencyGiven_ShouldPrepareWithOne() {
+    // Arrange
+    configureBehavior(null);
+    get = prepareGetWithClusteringKey();
+    get.withConsistency(Consistency.EVENTUAL);
+
+    // Act
+    handler.setConsistency(builder, get);
+
+    // Assert
+    verify(builder).setConsistencyLevel(ConsistencyLevel.ONE);
+  }
+
+  @Test
+  public void
+      setConsistency_GetOperationWithLinearizableConsistencyGiven_ShouldPrepareWithSerial() {
+    // Arrange
+    configureBehavior(null);
+    get = prepareGetWithClusteringKey();
+    get.withConsistency(Consistency.LINEARIZABLE);
+
+    // Act
+    handler.setConsistency(builder, get);
+
+    // Assert
+    verify(builder).setConsistencyLevel(ConsistencyLevel.SERIAL);
+  }
+
+  @Test
+  public void setConsistency_ScanOperationWithStrongConsistencyGiven_ShouldPrepareWithQuorum() {
+    // Arrange
+    configureBehavior(null);
+    scan = prepareScan();
+    scan.withConsistency(Consistency.SEQUENTIAL);
+
+    // Act
+    handler.setConsistency(builder, scan);
+
+    // Assert
+    verify(builder).setConsistencyLevel(ConsistencyLevel.QUORUM);
+  }
+
+  @Test
+  public void setConsistency_ScanOperationWithEventualConsistencyGiven_ShouldPrepareWithOne() {
+    // Arrange
+    configureBehavior(null);
+    scan = prepareScan();
+    scan.withConsistency(Consistency.EVENTUAL);
+
+    // Act
+    handler.setConsistency(builder, scan);
+
+    // Assert
+    verify(builder).setConsistencyLevel(ConsistencyLevel.ONE);
+  }
+
+  @Test
+  public void
+      setConsistency_ScanOperationWithLinearizableConsistencyGiven_ShouldPrepareWithSerial() {
+    // Arrange
+    configureBehavior(null);
+    scan = prepareScan();
+    scan.withConsistency(Consistency.LINEARIZABLE);
+
+    // Act
+    handler.setConsistency(builder, scan);
+
+    // Assert
+    verify(builder).setConsistencyLevel(ConsistencyLevel.SERIAL);
+  }
+
+  @Test
+  public void handle_DriverExceptionThrown_ShouldThrowProperExecutionException()
```

Comment:
Unless I am missing something I think it should be:

```suggestion
  public void handle_DriverExceptionThrown_ShouldThrowExecutionException()
```

Reply:
Thanks! Fixed.